app/models/batch.rb

```diff
@@ -477,13 +476,17 @@ def swap(current_user, batch_info = {}) # rubocop:todo Metrics/CyclomaticComplex
       batch_request_left.batch.lab_events.create!(
         description: 'Lane swap',
         message:
+          # rubocop:todo Layout/LineLength
```

Comment:
Same comment about moving this up a line (or in this case indenting the enable line if that makes more sense)